Doc/library/stdtypes.rst

```diff
@@ -3520,12 +3520,11 @@ copying.
    is a single byte, but other types such as :class:`array.array` may have
    bigger elements.
 
-   ``len(view)`` is equal to the length of :class:`~memoryview.tolist`.
-   If ``view.ndim = 0``, the length is 1. If ``view.ndim = 1``, the length
-   is equal to the number of elements in the view. For higher dimensions,
-   the length is equal to the length of the nested list representation of
-   the view. The :class:`~memoryview.itemsize` attribute will give you the
-   number of bytes in a single element.
+   ``len(view)`` is equal to the length of :class:`~memoryview.tolist`, which
```

Comment:
Since the documented behavior was changed, it needs a `versionchanged` directive and an entry in What's New.

Reply:
Should I put this in the 3.9 what's new? This sounds like a recipe for merge conflicts, until the rest of the patch has general approval.

Reply:
Just checking in on this - what's the procedure for adding a what's new entry if I have no idea which python version this will end up being merged into?

Reply:
You can use version `3.10`. It cannot go into earlier versions since it's a behavior change.  And we have ample time for deliberation on how the documented behavior emerged.

Reply:
I've update it to say `3.11`, since it's been two years since I changed it to say `3.10`. Let me know if it should be `3.12` instead.